_security/configuration/disable-enable-security.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: Disabling and enabling the Security plugin
+parent: Configuration
+nav_order: 40
+has_toc: true
+redirect_from: 
+ - /security-plugin/configuration/disable/
+---
+
+# Disabling and enabling the Security plugin
+
+If you're interested in using OpenSearch without security to make testing or internal usage more straightforward, you can temporarily disable the Security plugin. OpenSearch uses two separate security plugins, one for OpenSearch (Security plugin) and one of OpenSearch Dashboards (OpenSearch Dashboards Security plugin) You can use the OpenSearch by [disabling the Security plugin](#disabling-security) or [uninstalling the plugin](#removing-the-security-plugin).  However by default OpenSearch Dashboards, requires a secured OpenSearch cluster, which means that the OpenSearch Dashboards Security plugin must be [uninstalled](#removing-the-opensearch-dashboards-security-plugin) from OpenSearch Dashboards in order to use Dashboards without security.
```

Comment:
I don't think my second paragraph is technically correct, but I'm trying to make sense of the functionality. :)

Reply:
I'm not sue if 'permanent' is suitable here, since the plugin can of course be reinstalled at a later stage.
Perhaps: "If you want a more longer term option, you can remove the plugin completely, you can install the security plugin again when you are ready to secure the cluster."

Reply:
Reasons for uninstalling OpenSearch core:

- If you have your own security solution. 
- If you need to uninstall plugin for development purposes.

I'll add this detail in the next pass.

Reply:
If you remove one Security plugin from OpenSearch core, you have to remove it from OpenSearch Dashboards.